src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -429,14 +449,40 @@ class SeekBar extends Slider {
    * Move more quickly fast forward for keyboard-only users
    */
   stepForward() {
-    this.userSeek_(this.player_.currentTime() + this.options().stepSeconds);
+    // if `disableSeekWhileScrubbingOnSTV: true`, keep track of the desired seek point but we won't initiate the seek
+    if (this.shouldDisableSeekWhileScrubbing_) {
```

Comment:
There seems to be a bit of duplicate code here and below. Could we break some of this out into a helper `getCurrentPos` method or something like that, and then reuse it?